bin/phpstan/functions.stub

```diff
@@ -0,0 +1,38 @@
+<?php
+/**
+ * Function stubs for PHPStan static analysis.
```

Comment:
Does PHPStan require us to duplicate these, or can we configure them to be ignored since they are polyfilled in WP Core?

Reply:
I'm not entirely clear why these have to be duplicated. I think it's because wp-stubs doesn't include polyfilled functions? See https://github.com/php-stubs/wordpress-stubs/issues/100

Reply:
I‘d just ignore the errors instead, personallly

Reply:
@swissspidy What about 828c87f?

The only issue I've seen with ignoring errors is that it causes problems when there are ignored errors which are not present in the subset of files being analyzed. Then you have to add `reportUnmatchedIgnoredErrors: false`. Clearly this doesn't matter if these two PHP8 functions are the only errors being ignored, but if you ignore other errors which no longer are present anymore then they could get ignored without being present.

Reply:
I don't fully understand the problem you are describing about `reportUnmatchedIgnoredErrors`, but https://github.com/WordPress/performance/commit/828c87fef5154e9207502ed3842c2b535018fa64 works for me.